src/components/Day.vue

```diff
@@ -6,7 +6,7 @@
       @click='dayClicked(date)'
       v-text='`${dayNumber}`'
       :class='dayClass'
-      :style='isToday ? "border: 1px solid #00c690" : ""'
+      :style='isToday ? setcurrentDateStyle : ""'
```

Comment:
I think we don't need this `serCurrentDateStyle` computed prop. We can use the `currentDateStyle` directly here: `:style='isToday ? currentDateStyle : ""'`

Reply:
@krystalcampioni , Yes you're right, i have updated my code please check and let me know